src/hotspot/share/runtime/continuationEntry.cpp

```diff
@@ -33,13 +33,11 @@
 #include "runtime/thread.inline.hpp"
 
 int ContinuationEntry::return_pc_offset = 0;
-nmethod* ContinuationEntry::continuation_enter = nullptr;
 address ContinuationEntry::return_pc = nullptr;
 
-void ContinuationEntry::set_enter_nmethod(nmethod* nm) {
+void ContinuationEntry::set_enter_nmethod(CompiledMethod* cm) {
```

Comment:
This looks weird. Do we really have `CompiledMethod*` arguments for this call? If so, it would be weird to call it `set_enter_*nmethod*`.

Reply:
`CompiledMethod` is all that's required here, and I also want to try calling that method from elsewhere (Method::set_code) where we only have `CompiledMethod`. But if it bothers anyone I can leave it an nmethod.

Reply:
I agree, maybe rename to set_enter_method or set_enter_code() ?
Also the static data members don't have underscores, so I had to look to see what return_pc is, ie what this function did.  Was there a reason? If not, please add leading underscores to these nonstatic data members.

Reply:
Done